### ISSUE TITLE
Home Screen - Small Event Cards

### ISSUE THREAD

**Issue by xavjones8** [NCSU-App-Development-Club/Marigold-Android#8]
# **Description**:

Design hardcoded small event cards. Create a new composable files for each Figma component in a new "home" package.

# **Accessibility Requirements**:
Text and content should be readable at maximum 200% screen zoom.
The accessibility label should announce the card as a button, and read off all the text in the card.

# **Links and Resources**:

Draft designs can be found here under the **************Xavier************** tab, but are not finalized

https://www.figma.com/file/raH7fqviVPnDFKILPBJ5iN/NCSU-Club-%26-Events-App?type=design&node-id=0-1&mode=design&t=vAJwOwAhwt5KSvGg-0

# **Attatchments**:
![EventCard-1](https://github.com/NCSU-App-Development-Club/Marigold-Android/assets/44850083/a7179357-4b45-449a-a024-c57f07bac9ae)
